bugsnag-android-core/src/test/java/com/bugsnag/android/SessionSerializationTest.kt

```diff
@@ -0,0 +1,27 @@
+package com.bugsnag.android
+
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.Parameterized
+import org.junit.runners.Parameterized.Parameter
+import org.junit.runners.Parameterized.Parameters
+import java.util.Date
+
+@RunWith(Parameterized::class)
+internal class SessionSerializationTest {
+
+    companion object {
+        @JvmStatic
+        @Parameters
+        fun testCases() = generateSerializationTestCases(
+            "session",
+            Session("123", Date(0), User(null, null, null), 1, 0)
```

Comment:
I'd rather `User` had some non-null data to make sure the object is used in the output.

Reply:
Added a placeholder value for User to ensure it is serialized in the session JSON.